app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
```

Comment:
I'm trying to sync the format. Under "Terms", the "Show" link is linked to the term name. It makes sense to do the same for "Users".
![screen shot 2017-07-04 at 15 31 38](https://user-images.githubusercontent.com/15064985/27841289-01f18506-60ce-11e7-91c0-8e5fb591b5b0.png)